src/components/layouts/Layout.js

```diff
@@ -40,38 +40,54 @@ const Layout = ({ children, initialContext, hasSideBar, location }) => {
   const [context, setContext] = useState(initialContext || 'homepage');
   const [menuVisible, setMenuVisible] = useState(false);
   const lastPartofPath = /[^/]*$/;
-  const currentPagePath = location ? location.pathname.match(lastPartofPath) : 'Home';
+  const currentPagePath = location
+    ? location.pathname.match(lastPartofPath)
+    : 'Home';
   const currentPageName = currentPagePath
     .toString()
     .replace(/-/g, ' ')
     .split(' ')
-    .map(word => word.charAt(0).toUpperCase() + word.substring(1))
+    .map((word) => word.charAt(0).toUpperCase() + word.substring(1))
     .join(' ');
 
   return (
     <StaticQuery
       query={graphql`
-      {
-        allSparkIconSet {
-          edges {
-            node {
-              internal {
-                content
+        {
+          allSparkIconSet {
+            edges {
+              node {
+                internal {
+                  content
+                }
               }
             }
           }
         }
-      }
-    `}
-      render={data => (
+      `}
+      render={(data) => (
         <>
-          <div className="sprk-o-Box docs-c-SkipNav">
+          <div className="sprk-o-Box docs-c-SkipNav sprk-u-BackgroundColor--white">
```

Comment:
why did you update the background color here?

Reply:
original PR was made in the wrong branch, background color change is not part of this task, v13 already had background color set to white  https://github.com/sparkdesignsystem/spark-design-system/blob/8ca75e2b24930742be3621ce5f8d9e84bdbda962/src/components/layouts/Layout.js#L68